Dockerfile

```diff
@@ -0,0 +1,23 @@
+﻿FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build
+
+WORKDIR /src
+
+COPY PKHaX.csproj .
+
+RUN dotnet restore "PKHaX.csproj"
+
+COPY . .
+
+ARG ARCHITECTURE=x64
+RUN dotnet publish "PKHaX.csproj" -c Release -o /app/publish -r "linux-$ARCHITECTURE"
+
+FROM debian:12
+WORKDIR /app
+
+ENV PKHAX_PORT=9000
+ENV PKHAX_PRIVATE_KEY_PATH=/app/private.key
+EXPOSE 900

```

Comment:
Should this be 9000? Or was this intentional?

Reply:
Good catch, that should be 9000